src/TodoModel.cpp

```diff
@@ -20,6 +20,12 @@ TodoModel::TodoModel() {
     this->description = "Description ...";
     this->status = false;
 }
+TodoModel::TodoModel(int id) {
```

Comment:
Yeah so, in a nutshell,
A TodoModel can only be able to be constructed if and only if an `id` and `title` is supplied. but, we should be happy to receive `description` and `status` but its not mandatory. The default values for `description` and `status` if not supplied should be `""` and `false` respectively.

Make appropriate changes in the header file too.